tests/neo-vm.Tests/Tests/OpCodes/Arrays/VALUES.json

```diff
@@ -0,0 +1,194 @@
+        "NEWMAP",
+        "DUP",
+        "DUP",
+        "PUSH0",
+        "PUSH2",
+        "SETITEM",
+        "PUSH1",
+        "PUSH4",
+        "SETITEM",
+        "VALUES"
+      ],
+      "steps": [
+        {
+          "actions": [
+            "execute"
+          ],
+          "result": {
+            "state": "HALT",
+            "resultStack": [
+              {
+                "type": "array",
+                "value": [
+                  {
+                    "type": "Integer",
+                    "value": 2
+                  },
+                  {
+                    "type": "Integer",
+                    "value": 4
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "name": "Simple values in array",
+      "script": [
+        "PUSH2",
+        "NEWARRAY_T",
+        "0x00",
+        "VALUES"
+      ],
+      "steps": [
+        {
+          "actions": [
+            "execute"
+          ],
+          "result": {
+            "state": "HALT",
+            "resultStack": [
+              {
+                "type": "array",
+                "value": [
+                  {
+                    "type": "Null"
+                  },
+                  {
+                    "type": "Null"
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "name": "Compound value in array [inner struct]",
+      "script": [
+        "PUSH1",
+        "NEWARRAY",
+        "DUP",
+        "PUSH0",
+        "PUSH2",
+        "NEWSTRUCT",
+        "SETITEM",
+        "VALUES"
+      ],
+      "steps": [
+        {
+          "actions": [
+            "execute"
+          ],
+          "result": {
+            "state": "HALT",
+            "resultStack": [
+              {
+                "type": "array",
+                "value": [
+                  {
+                    "type": "struct",
+                    "value": [
+                      {
+                        "type": "Null"
+                      },
+                      {
+                        "type": "Null"
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "name": "Compound value in array [inner map]",
+      "script": [
+        "PUSH1",
+        "NEWARRAY",
+        "DUP",
+        "PUSH0",
+        "NEWMAP",
+        "DUP",
+        "PUSH0",
+        "PUSH1",
+        "SETITEM",
+        "SETITEM",
+        "VALUES"
+      ],
+      "steps": [
+        {
+          "actions": [
+            "execute"
+          ],
+          "result": {
+            "state": "HALT",
+            "resultStack": [
+              {
+                "type": "array",
+                "value": [
+                  {
+                    "type": "map",
+                    "value": {
+                      "": {
```

Comment:
fyi; this odd key value (`""`) is because an `Integer` stackitem with a 0 value returns an empty byte array instead of a byte array with a single item set to 0, see
https://github.com/neo-project/neo-vm/blob/754bca3568c50695c571cb796c500fd66c07d0fe/src/neo-vm/Types/Integer.cs#L16
It therefore translates to `""` instead of `0x00` as all other values do.